bindings/c/test/apitester/TesterTestSpec.cpp

```diff
@@ -47,7 +47,7 @@ std::unordered_map<std::string, std::function<void(const std::string& value, Tes
 	  } },
 	{ "apiVersion",
 	  [](const std::string& value, TestSpec* spec) { //
-	      processIntOption(value, "apiVersion", spec->apiVersion, 700, 710);
+	      processIntOption(value, "apiVersion", spec->apiVersion, 710, 720);
```

Comment:
I think maybe the first value shouldn't change, is that correct @sfc-gh-vgasiunas?

Reply:
set the first one back to `700` will leave this comment open until we get confirmation.

Reply:
It is ok. For the current tests it does not matter, and for upgrade testing I am going to reduce this value to 400 (the minimal version supported by MVC)